City-of-Helsinki/helsinki-paatokset
# PP-189: Add checkbox field to hide decision node content

**To test**
- Checkout branch and run `make drush-cr drush-cim`
- Check that the hidden decision message is editable and translatable at: https://helsinki-paatokset.docker.so/fi/admin/tools/default-texts
  - Add a custom message and a swedish translation
  - Run `make drush-cim` again. The default texts should be in config ignore, so they should not get overridden.
- Edit a decision: https://helsinki-paatokset.docker.so/fi/admin/content/decisions
  - Try to find one with a diary number that has other decisions so you can test the dropdown
  - Toggle the "Hide decision content" selection
  - Open the decision. You should see the message you added instead of the decision's content
  - Change the language to swedish. You should see the translated message
  - Navigate to other decisions using the dropdown. The content should show up normally for them, but when you select the one you edited you should only see the message.
